config/src/lib.rs

```diff
@@ -298,6 +298,8 @@ impl Parameters {
 
 #[derive(Clone, Serialize, Deserialize, Eq, Hash, PartialEq, Debug)]
 pub struct WorkerInfo {
+    /// The public key of this worker.
+    pub name: PublicKey,
```

Comment:
I think this new field breaks sui genesis, where we construct such a `WorkerInfo`